source/atlas-cli-changelog.txt

```diff
@@ -14,6 +14,32 @@
    :depth: 1
    :class: singlecol
 
+.. _atlas-cli-1.12.2:
+
+{+atlas-cli+} 1.12.2
+--------------------
+
+Released 20 October 2023
+
+- Adds an unsupported distro warning if you try to run local
+  deployment commands on an unsupported operating system. 
+- Adds `Podman <https://podman.io/>`__ as a recommended dependency for
+  the ``deb`` and ``rpm`` meta packages.
+- Adds :ref:`atlas` and :ref:`atlas-deployments` to
```

Comment:
```suggestion
- Adds listing of :ref:`atlas` and :ref:`atlas-deployments` to
```

- also not sure how we want to capture "local" deployments here. 
Context: this change means that now the deployments commands interactive flow will allow users to pick either a local or atlas cluster